test/abstract/SignContextAbstractTest.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: CAL
+pragma solidity ^0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+
+import {ECDSAUpgradeable as ECDSA} from "openzeppelin/utils/cryptography/ECDSAUpgradeable.sol";
+import {SignedContextV1} from "rain.interpreter.interface/interface/IInterpreterCallerV2.sol";
+
+abstract contract SignContextAbstractTest is Test {
```

Comment:
this seems like it could just be a lib rather than something that requires inheritance to use